packages/freezed/lib/src/freezed_generator.dart

```diff
@@ -398,6 +403,7 @@ Read here: https://github.com/rrousselGit/freezed/tree/master/packages/freezed#t
 
     return Freezed(
       unionKey: rawUnionKey.replaceAll("'", r"\'").replaceAll(r'$', r'\$'),
+      fallbackUnion: fallbackUnion,
```

Comment:
We'll need error handling, for cases where the `fallbackUnion ` key is specified but doesn't match a class constructor.

Throwing an `InvalidGenerationSourceError` with the error message would make sense.
It'll need a test, which you can test inside `test/source_gen_test.dart`/`source_gen_src.dart`

Reply:
done